docs/charts/mermaid/MongoAPIError.mmd

```diff
@@ -3,3 +3,5 @@ graph TD
     MongoAPIError --> MongoCompatibilityError
     MongoAPIError --> MongoMissingCredentialsError
     MongoAPIError --> MongoMissingDependencyError
+    MongoAPIError --> MongoExpiredSessionError
```

Comment:
I don't think the MongoAPI and MongoRuntime mermaids and svgs are needed in the repo anymore since the table is there.